src/main/java/PersonalBudget/business/user/domain/service/UserService.java

```diff
@@ -14,26 +16,38 @@
 
 @Service
 @RequiredArgsConstructor
-public class UserService implements UserDetailsService{
+public class UserService implements UserDetailsService {
 
     private final UserRepository userRepository;
-    private final IncomeCategoryRepository incomeCategoryRepository;
     private final UserMapper userMapper;
     private final BCryptPasswordEncoder bCryptPasswordEncoder;
 
     public boolean isEmail(UserDTO userDTO) {
-        if (userRepository.existsByEmail(userDTO.email())) {
-            return true;
-        } else {
-            return false;
-        }
+        return userRepository.existsByEmail(userDTO.email());
     }
 
     public void addNewUser(UserDTO userDTO) {
-        UserEntity userEntity = userMapper.mapUserDTOToUserEntity(userDTO, bCryptPasswordEncoder);
+        UserAccountEntity userEntity = userMapper.mapUserDTOToUserEntity(userDTO, bCryptPasswordEncoder);
         userRepository.save(userEntity);
-        Long userId = userRepository.findIdByEmail(userDTO.email());
-        incomeCategoryRepository.addDefaultCategories(userId);
+    }
+
+    public Long getNewUserId(String email) {
```

Comment:
Why is it a new user? The naming is a little confusing -> If the user is new how can we have it in the database?